diff --git a/setup.py b/setup.py
index 568b97f..c1d3c9e 100755
--- a/setup.py
+++ b/setup.py
@@ -69,9 +69,9 @@ if USE_CYTHON:
     ext = 'pyx'
     write_version(os.path.join(PKG_DIR, 'version.pxi'), {
         '__version__': VERSION,
-        '__git_version__': git_version(),
-        '__croaring_version__': chdir(git_tag, 'CRoaring'),
-        '__croaring_git_version__': chdir(git_version, 'CRoaring'),
+        '__git_version__': 'a9afead15d5f0be98d9edde440dc54167690beed',
+        '__croaring_version__': 'v0.2.66',
+        '__croaring_git_version__': '5f20740ec0de5e153e8f4cb2ab91814e8b291a14',
     })
 else:
     print('Building pyroaring from C sources.')
