diff --git a/setup.py b/setup.py
index 568b97f..c1d3c9e 100755
--- a/setup.py
+++ b/setup.py
@@ -69,9 +69,9 @@ if USE_CYTHON:
     ext = 'pyx'
     write_version(os.path.join(PKG_DIR, 'version.pxi'), {
         '__version__': VERSION,
-        '__git_version__': git_version(),
-        '__croaring_version__': chdir(git_tag, 'CRoaring'),
-        '__croaring_git_version__': chdir(git_version, 'CRoaring'),
+        '__git_version__': '4be9e068422b407b43589153e1ad370e060e5501',
+        '__croaring_version__': 'v0.3.1',
+        '__croaring_git_version__': '018c87d14e931a8d4dbab045e120161d9a660f76',
     })
 else:
     print('Building pyroaring from C sources.')
